core/src/main/java/org/apache/calcite/adapter/enumerable/RexToLixTranslator.java

```diff
@@ -856,11 +856,6 @@ public boolean isNullable(RexNode e) {
     return e.getType().isNullable();
   }
 
-  public RexToLixTranslator setNullable(
-      Map<? extends RexNode, Boolean> nullable) {
-    return this;
-  }
```

Comment:
Seems the nullable is never used ?

Reply:
Yes, it is useless in current algorithm.
Before CALCITE-3224, we need to maintain RexNode's nullable state in `RexToLixTranslator`.